sql/core/src/test/scala/org/apache/spark/sql/execution/command/DDLSuite.scala

```diff
@@ -93,7 +92,7 @@ class DDLSuite extends QueryTest with SharedSQLContext with BeforeAndAfterEach {
         .add("col2", "string")
         .add("a", "int")
         .add("b", "int"),
-      provider = Some("parquet"),
+      provider = Some("hive"),
```

Comment:
Actually it was a mistake. This suite needs to test `set location` or something which is only valid for hive tables. However, previously we only set the provider name in provider field, but not the table properties. Then in the commands we will mistakenly treat it as hive table and won't throw exception.

Now the hive hacks are all moved to external catalog, so setting provider name in provider field is enough. And we need to use `hive` for hive tables.